### DISCUSSION TITLE
单独升级electron版本应该不会有什么问题吧

### DISCUSSION THREAD

**Discussion by rkxie** [alex8088/electron-vite#507] [General]
需要做个多窗口，发现当前electron版本还不支持WebContentsView，能单独升级electron就好了
Note The BrowserView class is deprecated, and replaced by the new [WebContentsView]

**Comment by alex8088** [comment]
electron-vite  和 electron 版本没有强关系，所有版本都可以支持。自行升级electron版本就好
